components/Themes/src/MDCSemanticColorScheme.h

```diff
@@ -0,0 +1,80 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import "MDCColorScheme.h"
+
+/**
+ A simple color scheme that provides more semantic context for the colors it uses. Unlike
+ @c MDCColorScheme, there are no optional properties, so more consistent theming can be applied.
+ */
+@protocol MDCSemanticColorScheming <MDCColorScheme>
+
+/**
+ Displayed most frequently across your app.
+ */
+@property(nonnull, readonly, nonatomic) UIColor *primaryColor;
+
+/**
+ A tonal variation of primary color.
+ */
+@property(nonnull, readonly, nonatomic) UIColor *primaryColorLightVariant;
+
+/**
+ A tonal variation of primary color.
+ */
+@property(nonnull, readonly, nonatomic) UIColor *primaryColorDarkVariant;
+
+/**
+ Accents select parts of your UI.
```

Comment:
> If null, it is expected that the primaryColor will be used instead.

Reply:
This is assuming the API is still nullable, pending the decision below.

Reply:
Nullability wouldn't matter. If we set the properties to readonly, nonnull, then the implementations will need to ensure that all of the required set-up is done in `init`.